lldb/source/Plugins/ExpressionParser/Swift/SwiftREPL.cpp

```diff
@@ -262,9 +263,8 @@ lldb::REPLSP SwiftREPL::CreateInstanceFromDebugger(Status &err,
   // Disable the cleanup, since we have a valid repl session now.
   cleanup.release();
 
-  std::string swift_full_version(swift::version::getSwiftFullVersion());
-  printf("Welcome to %s.\nType :help for assistance.\n",
-         swift_full_version.c_str());
+  printf("You are now in an interactive environment powered by the LLDB debugger where you can type, run, and debug Swift code.\n\n");
+  printf("Type \x1B[1m:help\x1B[0m for debugger assistance. Type \x1B[1m:exit\x1B[0m to quit.\n");
```

Comment:
I like this message. Maybe while we're at it, could we tell users that they can run any LLDB commands with `:command`? I feel that many users don't know that's possible,

I think we should definitely keep the swift version though, as that may be helpful to users.

Reply:
So the reason why I removed the version print is that I pushed it up to the driver so that it appears consistently at the top when running other sort of "automatic" commands. If the Swift REPL is no longer run by default (something I eventually want to change), then running the REPL will be an explicit command without the larger context. It wouldn't hurt to print the version twice in the meantime but that was the reason, anyway.